dev/com.ibm.ws.security.audit.source/src/com/ibm/ws/security/audit/encryption/AuditCrypto.java

```diff
@@ -1228,6 +1257,9 @@ static final byte[] signISO9796(byte[][] key, byte[] data, int off, int len, boo
                 Tr.debug(tc, "Error: Signature operation failed");
                 Tr.error(tc, "security.ltpa.noalgorithm", new Object[] { e });
                 com.ibm.ws.ffdc.FFDCFilter.processException(e, "com.ibm.ws.security.audit.AuditCryptoCrypto", "1312");
+            } catch (NoSuchProviderException e1) {
+                Tr.debug(tc, "Error: No Such Provider");
+                Tr.error(tc, "security.ltpa.noalgorithm", new Object[] { e1 });
```

Comment:
`security.ltpa.noalgorithm` message key is not in the AuditMessage.nlsprops file.  As such no translated message will come out.  Should this be an error?  If so should the message key be included in the nls file.

Reply:
Yes, and not just that one - all the message key that we use today do not have it in the AuditMessage.nlsprops.
We will fix all the audit message keys later.

Reply:
I don't think we need ffdc here but added anyway.